es-app/src/views/gamelist/ISimpleGameListView.cpp

```diff
@@ -129,6 +132,32 @@ bool ISimpleGameListView::input(InputConfig* config, Input input)
 		{
 			ViewController::get()->goToRandomGame();
 			return true;
+		}else if (config->isMappedTo("y", input))
+		{
+			if(Settings::getInstance()->getString("VirtualSystemsAuto").find("favorites") != std::string::npos)
+			{
+				FileData* cursor = getCursor();
```

Comment:
LGTM. In fact, this won't be "toggle favorites" for long. It will be "add game to collection (game, collection)".